diff --git a/get-config.js b/get-config.js
index caf78653fda1c3464ffb12ff10e19cd82fce644a..ddd3f4a229791a633668bd66d7fb3a63217b5e70 100644
--- a/get-config.js
+++ b/get-config.js
@@ -16,7 +16,9 @@ module.exports = async function getConfig(limitConfig, check, output) {
     bundle: true,
     minifyWhitespace: true,
     minifyIdentifiers: true,
-    minifySyntax: true
+    minifySyntax: true,
+    
+    platform: 'node'
   }
 
   return config